dev-libs/libuv/libuv-1.5.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools-utils eutils multilib multilib-minimal
```

Comment:
Please don't convert my ebuilds to use `autotools-utils`. It even introduced a bug.